zip-0000.rst

```diff
@@ -300,6 +300,53 @@ Each ZIP SHOULD have the following parts:
   consensus within the community and discuss important objections or
   concerns raised during discussion.
 
+  For longer ZIPs it can potentially be easier to have inline Rationale
+  subsections interspersed throughout the Specification part. When taking
+  this approach, the content of these subsections should be annotated
+  with HTML tags to make it collapsible (so the rationale is available
+  for review but doesn't get in the way of reading the specification).
+  ZIPs written in Markdown can use the following syntax (note the
+  newline after the ``<summary>`` tag)::
+
+    # Specification
+
+    ## Foobar
+
+    Important details.
+
+    <details>
+    <summary>
+
+    ### Rationale for foobar
+    </summary>
+
+    Important but hidden rationale!
+    </details>
+
+  ZIPs written in reStructuredText can use the following syntax::
+
+    Specification
+    =============
+
+    Foobar
+    ------
+
+    Important details.
+
+    Rationale for foobar
+    ''''''''''''''''''''
+
+    .. raw:: html
+
+       <details>
+       <summary>Click to show/hide</summary>
```

Comment:
This approach is necessary for rST because it inserts a `<section>` wrapping the heading and content, and trying to wrap the heading in a `<summary>` causes the nesting to become wonky and breaks documents with more than one inline Rationale section.